updated line 38, 115-121 based on v1.0 data entries

ColisaNo195 opened the pull request:
updated line 38, 115-121 based on v1.0 data entries

ColisaNo195 committed:
updated line 115-122 based on v1.0 spreadsheet

ColisaNo195 committed:
Update sampleDSL-master.json

updated line 38, 115-121 based on v1.0 data entries

daphnecys reviewed (changes requested):
Generally, this is quite a good effort, so thanks for this. Just note that on the "Definitions" tab in the [TIOF] Project Lockdown [I] Dataset Layer structure EN v1.0 spreadsheet, there are values defined for the various *Types. Please use the corresponding numeric values for fields like "SourceType" and "Status". Please update and submit the pull request again.

ColisaNo195 committed:
I updated the following entries based on Definition tab:
11 - DSL - Status
89 - PLDInfo - Status
116 - Entries - Status
117 - Entries - Source Type
132 - Categories - Category Status
143 - DP - DP Status

ColisaNo195 commented:
Thank you for the instruction! I have edited a few statuses to numerical values, based on Definitions tab.
11 - DSL - Status
89 - PLDInfo - Status
116 - Entries - Status
117 - Entries - Source Type
132 - Categories - Category Status
143 - DP - DP Status
